pkg/prove/proof.go

```diff
@@ -109,7 +108,6 @@ func txSharePosition(txs types.Txs, txIndex uint64) (startSharePos, endSharePos
 
 	startSharePos = txShareIndex(prevTxTotalLen)
 	endSharePos = txShareIndex(endOfCurrentTxLen)
-	fmt.Printf("prevTxTotalLen: %d, endOfCurrentTxLen: %d, startSharePos: %d, endSharePos %d, currentTxTotalLen %d\n", prevTxTotalLen, endOfCurrentTxLen, startSharePos, endSharePos, currentTxTotalLen)
```

Comment:
[informational] when you run `make test` the output gets polluted with many lines of the form

```log
ok  	github.com/celestiaorg/celestia-app/pkg/inclusion	(cached)
prevTxTotalLen: 0, endOfCurrentTxLen: 82, startSharePos: 0, endSharePos 0, currentTxTotalLen 82
prevTxTotalLen: 82, endOfCurrentTxLen: 172, startSharePos: 0, endSharePos 0, currentTxTotalLen 90
prevTxTotalLen: 172, endOfCurrentTxLen: 562, startSharePos: 0, endSharePos 1, currentTxTotalLen 390
prevTxTotalLen: 562, endOfCurrentTxLen: 697, startSharePos: 1, endSharePos 1, currentTxTotalLen 135
prevTxTotalLen: 697, endOfCurrentTxLen: 956, startSharePos: 1, endSharePos 1, currentTxTotalLen 259
prevTxTotalLen: 956, endOfCurrentTxLen: 1004, startSharePos: 1, endSharePos 2, currentTxTotalLen 48
prevTxTotalLen: 1004, endOfCurrentTxLen: 1081, startSharePos: 2, endSharePos 2, currentTxTotalLen 77
prevTxTotalLen: 1081, endOfCurrentTxLen: 1350, startSharePos: 2, endSharePos 2, currentTxTotalLen 269
prevTxTotalLen: 1350, endOfCurrentTxLen: 1481, startSharePos: 2, endSharePos 2, currentTxTotalLen 131
prevTxTotalLen: 1481, endOfCurrentTxLen: 1635, startSharePos: 2, endSharePos 3, currentTxTotalLen 154
prevTxTotalLen: 1635, endOfCurrentTxLen: 1818, startSharePos: 3, endSharePos 3, currentTxTotalLen 183
...
```